flask/app/families.py

```diff
@@ -10,6 +11,21 @@
 @app.route("/api/families", methods=["GET"])
 @login_required
 def list_families():
+    starts_with = request.args.get("starts_with", default="", type=str)
+    starts_with = f"{starts_with}%"
+    max_rows = request.args.get("max_rows", default=100)
+    order_by_col = request.args.get("order", default="family_id", type=str)
+    try:
+        int(max_rows)
+    except:
+        return "Max rows must be a valid  integer", 400
```

Comment:
extra space

Reply:
resolved in [ae207ab](https://github.com/ccmbioinfo/ST2020/pull/195/commits/ae207abacd76978cf9409db8654f61ca59f5a1c5)